doc/installation.rst

```diff
@@ -264,6 +265,19 @@ In summary:
 
    This installs all of Spyder's dependencies into the environment along with the stable/packaged version of Spyder, and then removes Spyder itself.
 
+   .. note::
+
+      Following the separation of ``spyder-kernels`` into its own modular package in Spyder 3.3.0, you'll need to have the corresponding version of that package available—``0.x`` for Spyder 3 (``3.x`` branch), and ``1.x`` for Spyder 4 (``master`` branch).
+      The above procedure will install the ``0.x`` version; to test the ``master`` branch (Spyder 4), you'll need to install the corresponding ``1.x`` version of ``spyder-kernels``.
+      This can be done via two methods: installing the correct version via ``conda`` (or ``pip``):
+
+      .. code-block:: bash
+
+         conda install spyder-kernels=1.*
```

Comment:
This is not possible with `conda` because I instructed Anaconda guys to *not* package spyder-kernels >= 1.0.

So, this needs to be

    pip install --no-deps -U spyder-kernels

Reply:
Since the ``1.x`` version is available on ``conda-forge``, would ``conda install -c conda-forge spyder-kernels=1`` be better to suggest? That's how I have it now, and it seems to work just fine so far running from source. Is the risk of problems installing a package from a different channel with conda greater than using ``pip``? 

Seems a bit of inconvenience and potential problems for developers, but I assume its to avoid a (bigger) issue with users accidentally installing the wrong version. If using ``pip``, does the default conda-installed version also then need to be removed, in order to not have two versions?

Reply:
Also, once this gets cleared up, the Contributing Guide version needs to be fixed too since the problem wasn't caught there.

Reply:
UPDATE: Apparently, the 1.0.1 version is on conda defaults, at least according to my ``conda info spyder-kernels`` command.